portfolio/src/main/webapp/index.html

```diff
@@ -15,22 +15,48 @@
   </head>
   <body onload="getHelloMessage()">
     <div id="header">
-        <h1 id="helloheader"></h1>
+        <h1>Hi!</h1>
         <div id="tabs">
             <p id="aboutme-tab" class="tab" onclick="switchPage(this.id)">About Me</p>
             <p id="projects-tab" class="tab" onclick="switchPage(this.id)">Projects</p>
             <p id="pictures-tab" class="tab" onclick="switchPage(this.id)">Pictures</p>
             <p id="projectsexp-tab" class="tab" onclick="switchPage(this.id)">Projects 2.0</p>
+            <!-- <p id="comments-tab" class="tab" onclick="switchPage(this.id)">Comments</p> -->
         </div>
     </div>
     <div id="content">
+        <!-- <div id="comments-div" class="contentdiv" >
```

Comment:
Like the previous comment, this issue doesn't really matter for small projects/files, but I'll raise it here for educational purposes.

The state of the repo after a PR should generally be 'clean'. By this I mean removing large chunks of commented out code that don't contribute to the file's functionality or readability. If you'd like to preserve large chunks of code for future use or your own edification I'd suggest either:
1. Use the 'git stash' feature to store diffs locally.
2. Create a commit with the comment, and add an additional commit in the PR that removes the comment. Then, you can always bring back the comment via git commands (like `git show <commit_id>:<file_name>`).